javascript/exercises/objects.html

```diff
@@ -12,6 +12,35 @@
 <div class="container">
 <h2>Exercises: Objects</h2>
 
+<h3 class="page-header">Zookeeping</h3>
+
+<p><em>What kind of animals do you have in your zoo?!</em></p>
+
+<ul>
+ <li>Create an object to represent all of the <code>animals</code> you have at your zoo.</li>
+ <li>You can use <code>Object.keys(animals)</code> to get an array of animal types and then you can read the animal types using <code>console.log</code>.</li>
```

Comment:
Suggestion:
```
You can use <code>Object.keys(animals)</code> to get an array of animal types. Store the types in a variable called `animalTypes`.
```

By this time, students should know how to use `console.log()`, but sometimes they need explicit directions for how and where to store information